tests/test_api_hosts_update.py

```diff
@@ -271,6 +271,25 @@ def test_patch_produces_update_event_no_insights_id(
     )
 
 
+def test_patch_by_namespace_produces_update_event(
+    event_producer_mock, event_datetime_mock, db_create_host, db_get_host, api_patch
+):
+    host = db_host()
+    created_host = db_create_host(host=host, extra_data={"facts": DB_FACTS})
+
+    facts_url = build_facts_url(host_list_or_id=created_host.id, namespace=DB_FACTS_NAMESPACE)
+    response_status, response_data = api_patch(facts_url, DB_NEW_FACTS)
```

Comment:
Kind of a nitpick, but generally when a function returns a value that won't be used, you just use `_`.
```suggestion
    response_status, _ = api_patch(facts_url, DB_NEW_FACTS)
```

Reply:
I like to learn those things. I'll use _ from now on